site/_docs/reference.md

```diff
@@ -1200,7 +1200,7 @@ Note:
 | ANY      | The union of all types |
 | UNKNOWN  | A value of an unknown type; used as a placeholder |
 | ROW      | Row with 1 or more columns | Example: Row(f0 int null, f1 varchar)
-| MAP      | Collection of keys mapped to values |
+| MAP      | Collection of keys mapped to values | Example: MAP\['washington', 1, 'obama', 44\]
```

Comment:
This example is different from other examples.

I think the example should be `map(varchar, int)`.

Reply:
thanks. I have updated to map[varchar, int] instead of map(varchar, int), because the previous one is calcite standard form, the later one is Spark map form. We haven't implemented this yet. WDYT?

Reply:
@chucheng92  Do you know `generateTypeString`? Please take a look at them.
- ArraySqlType#generateTypeString
- MapSqlType#generateTypeString
- MultisetSqlType#generateTypeString

Reply:
@JiajunBernoulli thanks. Jiajun, I get your point. I actually wanted to add an example of map function at the beginning, here is actually map type. 

I have updated the PR. thanks for reminding me.

Reply:
@chucheng92 
In fact, we have many map constructor examples.

https://github.com/apache/calcite/blob/6f79436c178beec639e559d9152c237bbf8ec3e8/core/src/test/resources/sql/misc.iq#L2346

https://github.com/apache/calcite/blob/6f79436c178beec639e559d9152c237bbf8ec3e8/core/src/test/resources/sql/misc.iq#L2595